src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -96,14 +83,16 @@ public void clickSaveInstructorButton(int instrNum) {
         waitForPageToLoad();
     }
     
-    public String editInstructorName(String value) {
-        fillTextBox(editInstructorNameTextBox, value);
-        return getTextBoxValue(editInstructorNameTextBox);
+    public String editInstructorName(int instrNum, String value) {
+        WebElement editPanelNameTextBox = getNameField(instrNum);
+        fillTextBox(editPanelNameTextBox, value);
+        return getTextBoxValue(editPanelNameTextBox);
     }
     
-    public String editInstructorEmail(String value) {
-        fillTextBox(editInstructorEmailTextBox, value);
-        return getTextBoxValue(editInstructorEmailTextBox);
+    public String editInstructorEmail(int instrNum, String value) {
+        WebElement editPanelEmailTextBox = getEmailField(instrNum);
+        fillTextBox(editPanelEmailTextBox, value);
+        return getTextBoxValue(editPanelEmailTextBox);
     }
     
     public String fillInstructorName(String value) {
```

Comment:
I think this method is actually unused, can delete it, along with `instructorNameTextBox`